lib/membrane_rtsp/server/conn.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Membrane.RTSP.Server.Conn do
+  @moduledoc """
+  A module representing a client-server connection
+  """
+
+  use GenServer
+
+  import Membrane.RTSP.Server.Logic
+
+  require Logger
+
+  alias Membrane.RTSP.Server.Logic.State
+
+  @max_request_size 1_000_000
+
```

Comment:
This is just an arbitrary value. I thought 1 MB is enough for most requests, this number is too big since we don't support `ANNOUNCE` for now where the client will send a request body.
Do you have any suitable value in mind ?